internal/googlecloud/manufacturer.go

```diff
@@ -0,0 +1,25 @@
+//go:build !(linux || windows)
+
+/*
+ *
+ * Copyright 2021 gRPC authors.
```

Comment:
s/2021/2022 in this and the other new files

Reply:
Done.